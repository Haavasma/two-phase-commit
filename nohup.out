    Finished dev [unoptimized + debuginfo] target(s) in 0.01s
     Running `target/debug/orchestrator`
127.0.0.1 127.0.0.1 127.0.0.1 
127
[127, 0, 0, 1]
[127, 0, 0, 1]
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 98, kind: AddrInUse, message: "Address already in use" }', src/libcore/result.rs:1188:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
HOST:  127.0.0.1
Error listening:  listen tcp 127.0.0.1:3335: bind: address already in use
exit status 1
HOST:  127.0.0.1
Error listening: listen tcp 127.0.0.1:3332: bind: address already in use
exit status 1
