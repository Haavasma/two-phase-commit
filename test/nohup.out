/bin/bash: runservers: No such file or directory
/bin/bash: runservers: No such file or directory
/bin/bash: runservers: No such file or directory
